Contribution-Guidelines.md

```
@@ -9,7 +9,11 @@ We are excited about your interest in contributing to our web application projec
         
     - `dev`: The primary development branch where all features and bug fixes are integrated before merging into `master`. Always pull the latest `dev`before creating a new branch.
         
+<<<<<<< HEAD
```

Markdownlint (0.37.0)

Expected: atx; Actual: setext
Heading style

(MD003, heading-style)